tests/acceptance/enforce_one_basicget_test.py

```diff
@@ -0,0 +1,32 @@
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from mock import MagicMock
+from pika.frame import Method, Header
+from pika.exceptions import DuplicateGetOkCallback
+from pika.channel import Channel
+from pika.connection import Connection
+
+
+class OnlyOneBasicGetTestCase(unittest.TestCase):
```

Comment:
@dschep This is a unit test that uses mock, not an acceptance test. Therefore it belongs under tests/unit/, but not under tests/acceptance/, which establish live connections with the broker. Also, there is an existing unit test script for Channel unit tests, where these things belong: https://github.com/pika/pika/blob/master/tests/unit/channel_tests.py